app/controllers/project_database.py

```diff
@@ -193,7 +193,19 @@ def get(self, project_id, database_id):
         if errors:
             return dict(status='fail', message=errors), 500
 
-        return dict(status='success', data=dict(database=json.loads(database_data))), 200
+        # Check the database status on host
+        database_service = DatabaseService()
+        database_connection = database_service.create_db_connection(user=database.user, password=database.password, db_name=database.name)
```

Comment:
Same here, the connections have to be closed

Reply:
cool, let me work on it.

Reply:
Done